File: FrameworkLib_Common/Model/Club.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;

namespace FrameworkLib_Common.Model
{
    public class Club
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int Id { get; set; }

        public string Title { get; set; } = null!;
        public NotMappedText LocalizedText { get; set; } = null!;

        public virtual ICollection<Animal>?   Animals   { get; set; }
        public virtual ICollection<Location>? Locations { get; set; }
        public virtual ICollection<Grade>     Grades    { get; set; } = null!;
        public virtual ICollection<Drawback>? Drawbacks { get; set; }

        public override string ToString()
        {
            return $@"Club: Id = {Id} Title = {Title}";
        }
    }

    [NotMapped]
    public class NotMappedText
    {
        public string LocalizedText { get; set; } = null!;
    }
}

File: VBLib_Common/Model/Club.vb
Imports System.Collections.Generic
Imports System.ComponentModel.DataAnnotations
Imports System.ComponentModel.DataAnnotations.Schema

Namespace Model
    Public Class Club
        <Key>
        <DatabaseGenerated(DatabaseGeneratedOption.Identity)>
        Public Property Id As Integer
        Public Property Title As String
        Public Property LocalizedText As NotMappedText
        Public Overridable Property Animals As ICollection(Of Animal)
        Public Overridable Property Locations As ICollection(Of Location)
        Public Overridable Property Grades As ICollection(Of Grade)
        Public Overridable Property Drawbacks As ICollection(Of Drawback)

        Public Overrides Function ToString() As String
            Return $"Club: Id = {Id} Title = {Title}"
        End Function
    End Class

    <NotMapped>
    Public Class NotMappedText
        Public Property LocalizedText As String
    End Class
End Namespace
